config/samples/nim/guardrail/nimcache-content-safety.yaml

```
@@ -0,0 +1,19 @@
+---
+apiVersion: apps.nvidia.com/v1alpha1
+kind: NIMCache
+metadata:
+  name: content-safety-cache
+spec:
+  source:
+    ngc:
+      modelPuller: nvcr.io/nim/nvidia/llama-3.1-nemoguard-8b-content-safety:1.0.0
+      pullSecret: ngc-secret
+      authSecret: ngc-api-secret
+      model: {}
```

we can provide the model filter for this:
```suggestion
      model:
        engine: tensorrt_llm
        tensorParallelism: "1"
```

---

These params are not part of the nemoguard NIMs, we could have just said `all` instead here. But we have that issue where we don't update profiles in the status field for `all` .

---

No both content-safety and topic-control have these params. Only jailbreak-detect doesnt have it

---

Updated to use tp:1 wherever available, we can check to see if they can generate tensorrt engines with tp:1 for jailbreak too.